init.lua

```diff
@@ -17,6 +17,9 @@ dofile(MP.."/abm.lua")
 -- quarry
 dofile(MP.."/quarry.lua")
 
+-- switching station
+dofile(MP.."/quarry.lua")
```

Comment:
sure about this? 🤷

Reply:
:facepalm: ugh, thanks for looking over this